### DISCUSSION TITLE
wrong email address in openssl license

### DISCUSSION THREAD

**Discussion by hzhuang1** [openssl/openssl#23249] [General Discussion]
The email address of Andy is listed in the license. But when I sent email to him, I only got the error message in below.

The response from the remote server was:
551 5.7.1 <[appro@openssl.org](mailto:appro@openssl.org)>: Recipient address rejected: User has moved

**Comment by jamuir** [comment]
> The email address of Andy is listed in the license

Which file are you looking at?

I suspect the file is dual licensed.

**Comment by hzhuang1** [comment]
> > The email address of Andy is listed in the license
> 
> Which file are you looking at?
> 
> I suspect the file is dual licensed.

[email.log](https://github.com/openssl/openssl/files/13775321/email.log)

Log is attached.

Yes, the wrong email address is listed in those dual license.

**Comment by jamuir** [comment]
Sorry I wasn't more clear.  I wanted to know the particular source file you were interested in.  I realize that the email address is listed in many source files.

I guess that you emailed that address because you want to use the source file in another project.  I am not sure if that is required b/c the source file is also available under the Apache 2.0 license.

**Comment by hzhuang1** [comment]
> Sorry I wasn't more clear. I wanted to know the particular source file you were interested in. I realize that the email addresses is listed in many source files.
> 
> I guess that you emailed that address because you want to use the source file in another project. I am not sure if that is required b/c the source file is also available under the Apache 2.0 license.

I want to port aes-cbc algorithm from aesv8-armx.pl into linux kernel. I found that sha512-armv8.pl was ported from OpenSSL into linux kernel. So I want to do the similar thing on aes-cbc algorithm.

**Comment by tom-cosgrove-arm** [comment]
Related to #23156
